### ISSUE TITLE
Cannot read properties of undefined (reading 'browserHash')

### ISSUE THREAD

**Issue by bodograumann** [webfansplz/vite-plugin-vue-inspector#10]
I have just tried using the latest version 0.2.5, but got the following error:
```
3:26:36 PM [vite] Internal server error: Cannot read properties of undefined (reading 'browserHash')
      at transformIndexHtml (/home/bodo/proj/node_modules/vite-plugin-vue-inspector/dist/index.js:538:48)
      at applyHtmlTransforms (/home/bodo/proj/node_modules/vite/dist/node/chunks/dep-611778e0.js:27150:27)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async viteIndexHtmlMiddleware (/home/bodo/proj/node_modules/vite/dist/node/chunks/dep-611778e0.js:55903:28)
```

I am also using `vite-plugin-html-2.1.2`. Maybe there is a conflict?

**Comment by webfansplz** [comment]
Thanks for your feedback, Can you provide a mini repo ?

**Comment by lakca** [comment]
<img width="286" alt="截屏2022-04-05 下午6 10 06" src="https://user-images.githubusercontent.com/16255922/161731587-7c57f8ca-d7e0-4d05-91d7-154ef8d37bba.png">
<img width="609" alt="截屏2022-04-05 下午6 11 56" src="https://user-images.githubusercontent.com/16255922/161731810-aa9ecc8a-b15a-40b1-ad1a-d4630e7192fc.png">

@webfansplz 

relative code in vite has been refactored. [commit here](https://github.com/vitejs/vite/commit/676f54584c981c18413194bb539165292a66bf7c) & [issue](https://github.com/vitejs/vite/pull/7379)

**Comment by webfansplz** [comment]
Fixed in V0.3.0 !
